test_cli.py

```diff
@@ -1102,7 +1104,7 @@ def test(ctx):
             'ls dir; cat file; seq 1 10; touch done; while true; do sleep 60; done',
         ]
     )
-    wait_until_running(uuid)
+    wait_until_state(uuid)
```

Comment:
This call doesn't really make sense without `running` either in the function name or as an explicit argument.

Reply:
I relied on the default value too much...